docs/identity-oauth-providers.md

```
@@ -0,0 +1,336 @@
+    "Epic": {
+      "ClientId": "your-epic-client-id",
+      "ClientSecret": "your-epic-client-secret"
+    }
+  }
+}
+```
+
+### Redirect URI
+
+Configure in Epic Games portal:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)